frontend/src/components/pages/HomePage.tsx

```diff
@@ -19,14 +19,16 @@ const HomePage = () => {
     authenticatedUser!!.approvedLanguagesTranslation.replace(/'/g, '"'),
   );
 
+  const approvedLanguagesReview = authenticatedUser!!.approvedLanguagesReview
```

Comment:
it's to handle the case of `approvedLanguagesReview` being null, initially I tried doing something like 
`const approvedLanguagesReview = JSON.parse(authenticatedUser!!.approvedLanguagesReview.replace(/'/g, '"')) ?? {}` but it causes a TypeError because of the `.replace` call

Reply:
Ah ty for clarifying! make sense that there's a typescript error on the `.replace` call. Happy with the current code